apps/pushy/src/pushy_job_state.erl

```diff
@@ -115,12 +115,8 @@ init({#pushy_job{id = JobId, job_nodes = JobNodeList, opts = Opts} = Job, Reques
             lager:info([{job_id,Job#pushy_job.id}],
                         "Job ~p starting '~p' on ~p nodes, with timeout ~ps",
                         [JobId, Job#pushy_job.command, length(JobNodeList), Job#pushy_job.run_timeout]),
-
-            % Start voting--if there are no nodes, the job finishes immediately.
-            case start_voting(State) of
-                {next_state, StateName, State2} -> {ok, StateName, State2};
-                {stop, Reason, _State} -> {stop, Reason}
-            end;
+            {next_state, StateName, State2} = start_voting(State),
```

Comment:
This confused me first, but there seems to be no way `start_voting/1` would return `{stop, ...}` (as dialyzer probably pointed out?) -- so LGTM.